File: client/game/megaphone.ts
import { megaphone } from "../../shared/constants"
import { createMicFilter } from "../modules/megaphone"

setImmediate(() => {
  createMicFilter()
})

RegisterCommand(megaphone.command, () => {
  emit('lenix_patrolmegaphone:client:toggle')
}, false)

RegisterKeyMapping(megaphone.command, megaphone.description, 'keyboard', megaphone.key)

File: lenixm/lenix_patrol/client/game/megaphone.lua
local config<const> = require 'shared/constants/index'
local megaphoneMod<const> = require 'client/modules/megaphone'

SetTimeout(0, function()
  megaphoneMod.createMicFilter()
end)

RegisterCommand(config.megaphone.command, function()
  TriggerEvent('lenix_patrolmegaphone:client:toggle')
end, false)

RegisterKeyMapping(config.megaphone.command, config.megaphone.description, 'keyboard', config.megaphone.key)